WebRTCPlugin/WebRTCProxy.cpp

```diff
@@ -1,4 +1,3 @@
-

```

Comment:
general question: in case something goes wrong internally with the plugin functionality, how will be able to know what is wrong? we need some sort of log/callback solution. can you offer one please?

Reply:
For webrtc-based issues you should visit this for more info: https://webrtc.org/native-code/logging/
For the plugin, my only suggestion would be to create your own custom logging class, and/or use try/catch/throw.